server/tests/lib/nl/page_config_next_test.py

```diff
@@ -548,6 +549,93 @@
  }
 """
 
+EVENT_CONFIG = """
```

Comment:
add a TODO to save these textproto into test_data/*.textproto?

Reply:
Done.